tests/test_plugins_os_windows_tasks.py

```diff
@@ -40,7 +40,7 @@ def assert_xml_task_properties(xml_task):
     assert xml_task.user_id is None
     assert xml_task.logon_type is None
     assert xml_task.group_id == "S-1-5-4"
-    assert xml_task.display_name is None
+    assert xml_task.display_name is "test_xml.xml"
```

Comment:
```suggestion
    assert xml_task.display_name == "test_xml.xml"
```
Usually you only use a `is` comparison on things like `None` or class types. Any other type of comparison should be done with `==`.